web/src/app/models/survey.model.ts

```diff
@@ -30,15 +39,17 @@ export class Survey {
     /* jobs= */
     Map<string, Job>(),
     /* acl= */
-    Map<string, Role>()
+    Map<string, Role>(),
+    OnboardingStatus.NAME_SURVEY
   );
 
   constructor(
     readonly id: string,
     readonly title: string,
     readonly description: string,
     readonly jobs: Map<string, Job>,
-    readonly acl: Map<string, Role>
+    readonly acl: Map<string, Role>,
+    readonly onboardingStatus: OnboardingStatus
```

Comment:
If we make this nullable (null meaning "done") it'll be backwards compatible with existing surveys, and won't require DONE to always specified .

Reply:
Done.